sdk/formrecognizer/Azure.AI.FormRecognizer/src/Azure.AI.FormRecognizer.csproj

```diff
@@ -6,6 +6,8 @@
     <ApiCompatVersion>3.0.0</ApiCompatVersion>
     <PackageTags>Microsoft Azure Form Recognizer</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <!-- InheritDoc No matching non-duplicate nodes found -->
```

Comment:
Is the goal to eventually do something to get rid of this warning?

Reply:
Yes

Reply:
could you file issues for this? and do you have documentation that can be shared in the issues about what the problem is? or how to start addressing it?

Reply:
Actually - I decided to pivot the approach a bit - this PR will fix all non-generated warnings and I have a generator PR that will fix all the generated ones. I'll update this PR once [this](https://github.com/Azure/autorest.csharp/pull/954) goes through

Reply:
The generator update PR is here https://github.com/Azure/azure-sdk-for-net/pull/17245